json-formatter/ruby-testdata/Makefile

```diff
@@ -17,6 +17,10 @@ features/%.json: features/%-unprocessed.json
 features/rules/rules.feature.json:
 	mkdir -p $(@D)
 	echo "[]" > $@
+	
+features/retry/retry.feature.json:
```

Comment:
I put this in temporarily to get the build green, but just a little fuzzy on what I need to do to address it properly. Do I need to make changes to cucumber-ruby to add support code for the new CCK feature?

Reply:
Yes, I think that's what's needed.

Reply:
At the moment, the ruby-testdata are based on an old version of cucumber-ruby.

In the PR#1498 there is some changes to use current version of cucumber-ruby for those test data.
It has not been merged yet because of the discussion around the empty scenarios still being in progress.

Maybe we could consider merging PR#1498 - which add empty scenarios into the cck - and let implementations decide if they want to support it or not?

Reply:
Added the support code for ruby in cck and all green now.

Reply:
> As I understand, supporting `retry` won't be supported by all implementation either?

Yep, just Ruby and JavaScript AFAIK. Other impls will just need to filter this one out of their CCK setup.